packages/seacas/scripts/exodus3.in.py

```diff
@@ -851,12 +851,41 @@ def copy(self, fileName, include_transient=False, mode='a'):
                                           ctypes.byref(self.io_ws),
                                           EX_API_VERSION_NODOT)
 
-        self.__copy_file(fileId, include_transient)
+        self.copy_file(fileId, include_transient)
         EXODUS_LIB.ex_close(fileId)
 
         return exodus(fileName, mode)
 
 
+    #
+    # copy to a new already created database
+    #
+    # --------------------------------------------------------------------
+    def copy_file(self, file_id, include_transient=False):
+        """
+        Copies exodus database to the database pointed to by `fileId`
+        Does not return anything.
```

Comment:
This says that it does not return anything but it also says that it returns the file_id below. I think this is a typo.

Reply:
Yes, I changed that for some reason and didn't update the comment.  Not sure if returning `file_id` gives any benefit, but I added it from some reason...   Will update the comment.